src/main/java/com/twitter/http2/HttpConnectionHandler.java

```diff
@@ -837,19 +837,16 @@ public void write(ChannelHandlerContext ctx, Object msg, ChannelPromise promise)
                 return;
             }
 
-            // TODO(jpinner) handle push promise frames
```

Comment:
Looks good to me.  No obvious unit tests covering this, but tested in place.

If the fix was this simple, I'm curious why it wasn't enabled before?